stories/locfeature.HYPER.mdx

```diff
@@ -33,7 +33,7 @@ import contentArray from './locfeature.HYPER/carousel_content.json';
     <Embed
       style={{
         width:"100%",
-        height:"calc(95vw * 9/32)"
+        height:"calc(85vw * 9/32)",
```

Comment:
```suggestion
        height:"calc(((100vw - 48px) * 0.28) + 64px)",
        maxHeight: "calc(((1375px - 48px) * 0.28) + 64px)"
```

Can you try this and see if it works better? (64px is the height of the top navigation that doesn't seem to change ratio, 48px is a general padding value for the dashboard contents - so we can get an approximate value of the width by subtracting this number from 100vw. We put the limitation on how wide one block can stretch (which is approximately 1375px), so we need a max-height too.

Reply:
(Even if this value doesn't work, it would be a good idea to give some `maxHeight` attribute so the height doesn't expand on the wide screen.